configure.ac

```diff
@@ -2133,7 +2133,9 @@ fi
     fi
     RUST_SURICATA_LIB="${RUST_SURICATA_LIBDIR}/${RUST_SURICATA_LIBNAME}"
 
-    CFLAGS="${CFLAGS} -I\${srcdir}/../rust/gen -I\${srcdir}/../rust/dist -I../rust/gen"
+    # make sure that we use our headers for lua
+    CPPFLAGS="-I\${srcdir}/../rust/gen ${CPPFLAGS}"
+    CFLAGS="${CFLAGS} -I\${srcdir}/../rust/dist -I../rust/gen"
```

Comment:
I think all these -I for our local directories should go in CPPFLAGS, its the correct place for these paths.  And will ensure they all come before the others.

Reply:
oh thanks